mandatory/3-function-output.js

```diff
@@ -2,13 +2,16 @@
 function getRandomNumber() {
   return Math.random() * 10;
 }
+//A floating-point, pseudo-random number between 0 (inclusive) and 1 (exclusive)
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
-  return word1.concat(word2);
+  return word1.concat("", word2);
```

Comment:
this addition of a empty string as the first parameter of concat is un-necessary. `word1.concat(word2)` works fine.